labs/lab3-paxos/tst/dslabs/paxos/PaxosTest.java

```diff
@@ -241,6 +241,12 @@ private static StatePredicate hasCommand(Address a, int i, Command c) {
                         a, s, i, c));
             }
 
+            if ((s == ACCEPTED || s == CHOSEN) && c == null) {
```

Comment:
I'm not going to merge this check. Sometimes, people use `null` as a placeholder for a no-op in the log, which should be allowed.

Reply:
OK, that makes sense. I'll remove the check and backport it to our personal repo for testing.